pages/lobby_page.py

```diff
@@ -0,0 +1,139 @@
+
+    def find_input_alias(self):
+        self.context.wait.until(EC.presence_of_element_located((By.CSS_SELECTOR, '.notification.msg-line')))
+        return self.context.driver.find_elements_by_css_selector('.notification.msg-line')
+
+    def get_text_from_alias_bot(self):
+        for n in self.find_input_alias():
+            print(n.text)
+            for word in n.text.split():
+                if '@gtest' in word:
+                    return True
+
+    def chat_adding_alias(self):
+        self.input_comands_in_field()
+        return self.get_text_from_alias_bot()
+
+    def delete_alias_in_chat_comands(self):
+        self.find_input_field().send_keys('/alias remove @gtest')
+        self.find_input_field().send_keys(Keys.ENTER)
+
+    def random_click(self):
+        self.context.wait.until(EC.presence_of_element_located((By.ID, 'status_dropdown')))
+        self.find_element_for_random_click().click()
+
+    def find_element_for_random_click(self):
+        self.context.wait.until(EC.presence_of_element_located((By.XPATH, "//input[@placeholder='Filter']")))
+        return self.context.driver.find_element_by_xpath("//input[@placeholder='Filter']")
+
+    def open_alias_menu(self):
+        self.find_plus_button().click()
+
+    def click_for_focus_near_plus_button(self):
+        self.context.wait.until(EC.presence_of_element_located((By.CSS_SELECTOR, 'textarea#hc-message-input')))
+        self.context.driver.find_element_by_css_selector('textarea#hc-message-input').click()
+
+    def find_plus_button(self):
+        self.click_for_focus_near_plus_button()
+        self.context.wait.until(EC.presence_of_element_located(
+            (By.CSS_SELECTOR, 'div.hc-dropdown>button#input_actions_dropdown-trigger')))
+        return self.context.driver.find_element_by_css_selector('div.hc-dropdown>button#input_actions_dropdown-trigger')
+
+    def open_menu(self):
+        self.find_button_in_dropdown_menu().click()
+
+    def find_button_in_dropdown_menu(self):
+        self.context.wait.until(EC.presence_of_element_located((By.XPATH, ('//a[@data-addon_key="hc-alias"]'))))
+        return self.context.driver.find_element_by_xpath('//a[@data-addon_key="hc-alias"]')
+
+    def switch_to_frame(self):
+        time.sleep(3)
+        return self.context.wait.until(EC.frame_to_be_available_and_switch_to_it((By.CLASS_NAME, 'hc-addon-iframe')))
+
+    def open_config(self):
+        self.context.wait.until(EC.presence_of_element_located(
+            (By.CSS_SELECTOR, 'div>div>.aui-button.aui-button-link')))
+        self.context.driver.find_element_by_css_selector('div>div>.aui-button.aui-button-link').click()
+
+    def identify_frame_status_and_put_data(self):
+        try:
+            self.input_data_in_alias_form()
+            self.input_data_in_alias_name_form()
+            self.open_config()
+            self.input_data_in_alias_form()
+            self.input_data_in_alias_name_form()
+        except:
+            self.open_config()
```

Comment:
If just open_config() fails, shall we put self.input_data_in_alias_form() and self.input_data_in_alias_name_form() out of try clause?